docs/core/compendium_de.md

```diff
@@ -0,0 +1,1177 @@
+Wert `dogu` angenommen.
+
+Gültige Werte sind einer der folgenden: "dogu", "client", "package".
+
+Der Typ "dogu" verweist auf einen anderes Dogu, das während der Abhängigkeitsüberprüfung vorhanden und aktiv sein muss.
+
+Ein Typ "client" verweist auf den Client, der die "dogu.json" des Dogus verarbeitet. Mehrere Client-Abhängigkeiten eines
+anderen Client-Typs können z. B. verwendet werden, um die Verarbeitung eines bestimmten Clients zu verbieten.
+
+Ein Typ "package" verweist auf ein notwendiges Betriebssystem-Paket, das bei der Abhängigkeitsüberprüfung vorhanden sein
+muss.
+
+Beispiele:
+
+- "dogu"
+- "client"
+- "package"
+
+### Name
+
+Name identifiziert die durch den Typ ausgewählte Entität. Dieses Feld ist obligatorisch. Wenn der Typ eine andere
+Entität auswählt, muss Name den einfachen Entitätsnamen verwenden (z. B. "postgres"), nicht den voll qualifizierten
+Entitätsnamen (nicht "official/postgres").
+
+Beispiele:
+
+- "postgresql"
+- "k8s-dogu-operator"
+- "cesappd"
+
+### Version
+
+Version wählt die Version der durch den Typ ausgewählten Entität aus. Dieses Feld ist optional. Wenn es nicht gesetzt
+ist, wird jede Version der ausgewählten Entität bei der Abhängigkeitsüberprüfung akzeptiert.
+
+Version akzeptiert verschiedene Versionsstile und Vergleichsoperatoren.
+
+Beispiele:
+
+- ">=4.1.1-2" - akzeptiert größer oder gleich der Version 4.1.1-2
+- "<=1.0.1" - akzeptiert eine Entität, die kleiner oder gleich der Version 1.0.1 ist
+- "1.2.3.4" - wählt genau die Version 1.2.3.4 aus
+
+Mit einer nicht existierenden Version ist es möglich, eine Abhängigkeit zu negieren.
+
+Beispiel:
+
+- "<=0.0.0" - verbietet, dass die ausgewählte Entität vorhanden ist
+
+## type [Dogu](<https://github.com/cloudogu/cesapp-lib/blob/main/core/dogu_v2.go#L510-L951>)
+
+Dogu beschreibt die Eigenschaften einer containerisierten Anwendung für das Cloudogu EcoSystem. Neben den
+Meta-Informationen und dem [OCI Container Image](https://opencontainers.org/) beschreibt das Dogu alle Notwendigkeiten
+für die automatische Container-Instantiierung, z. B. Volumes, Abhängigkeiten zu anderen Dogus und vieles mehr.
+
+```go
+type Dogu struct {
+    Name string
+    
+    Version string
+    
+    DisplayName string
+    
+    Description string
+    
+    Category string
+    
+    Tags []string
+    
+    Logo string
+    
+    URL string
+    
+    Image string
+    
+    ExposedPorts []ExposedPort
+    
+    ExposedCommands []ExposedCommand
+    
+    Volumes []Volume
+    
+    HealthCheck HealthCheck
+    
+    HealthChecks []HealthCheck
+    
+    ServiceAccounts []ServiceAccount
+    
+    Privileged bool
+    
+    Configuration []ConfigurationField
+    
+    Properties Properties
+    
+    EnvironmentVariables []EnvironmentVariable
+    
+    Dependencies []Dependency
+    
+    OptionalDependencies []Dependency
+}
+```
+
+### Name
+
+Name enthält den voll qualifizierten Namen des Dogus, der sich aus dem Dogu-Namensraum und dem einfachen Namen des Dogus
```

Comment:
I think `namespace` should be left as-is because it pops up everywhere and stands for its own.